src/main/kotlin/com/tealcube/minecraft/bukkit/mythicdrops/api/items/CustomItem.kt

```diff
@@ -54,6 +54,7 @@ interface CustomItem : Weighted {
     val isEnchantmentsRemovableByGrindstone: Boolean
     val isAddDefaultAttributes: Boolean
     val hdbId: String
+    val rgb: String
```

Comment:
I would prefer this be represented as a full data structure instead of relying on parsing strings, which can get gross fairly quickly.

An example data structure would be:
```kotlin
data class Rgb(
  val red: Int,
  val green: Int,
  val blue: Int
) {
  fun toColor(): Color {
    return Color.fromRGB(red, green, blue)
  }
}
```